tools/advisor/advisor/db_config_optimizer.py

```diff
@@ -76,15 +79,26 @@ def improve_db_config(current_config, guidelines):
                     col_fam in current_config[option]
                 ):
                     old_value = float(current_config[option][col_fam])
-                new_value = ConfigOptimizer.apply_action_on_value(
-                    old_value, action, suggested_values
-                )
-                if new_value:
+                try:
+                    new_value = ConfigOptimizer.apply_action_on_value(
+                        old_value, action, suggested_values
+                    )
                     better_config[option][col_fam] = new_value
+                except AssertionError:
+                    print(
+                        'WARNING(ConfigOptimizer): provide suggested_values ' +
+                        'for ' + option
+                    )
         return better_config
 
     @staticmethod
     def improve_db_config_v2(options, rule, suggestions_dict):
+        '''
+        this method takes ONE 'rule' and applies all its suggestions on the
+        appropriate options whereas
+        'improve_db_config' picks ONE option to update and applies all
```

Comment:
we can remove the explanation about improve_db_config once we remove the function.